packages/react/package.json

```diff
@@ -37,19 +37,19 @@
     "@storybook/types": "7.6.20",
     "@types/react": "^18.3.4",
     "@types/react-dom": "^18.3.0",
-    "babel-loader": "^9.1.3",
+    "babel-loader": "^9.2.1",
     "dso-toolkit": "workspace:^",
-    "http-proxy-middleware": "^3.0.1",
-    "iframe-resizer": "^5.2.6",
+    "http-proxy-middleware": "^3.0.2",
+    "iframe-resizer": "^5.3.1",
```

Comment:
Waarom hebben we deze package eigenlijk nog 🤔 

Zou deze niet vervangen moeten zijn door `@iframe-resize/child` OF `@iframe-resize/parent` ?

> This package is a legacy version of [iframe-resizer](https://iframe-resizer.com/). This project has now been split in to separate packages for the parent and child pages.
> 
> [@iframe-resizer/parent](https://www.npmjs.com/package/@iframe-resizer/parent)
> [@iframe-resizer/child](https://www.npmjs.com/package/@iframe-resizer/child)

https://www.npmjs.com/package/iframe-resizer

Reply:
Ik heb hem weggehaald